share/ert/forward-models/shell/CAREFUL_COPY_FILE

```diff
@@ -0,0 +1,9 @@
+STDOUT      careful_copy_file.stdout
+STDERR      careful_copy_file.stderr
```

Comment:
The std file descriptors are redirected by default - not necessary to do it explicitly.